defaults/main.yml

```diff
@@ -6,9 +6,9 @@ RABBITMQ_ADMIN_PASSWORD: changeme
 RABBITMQ_TLS_PORT: 5671
 RABBITMQ_HTTPS_PORT: 15671
 
-RABBITMQ_CA_CERTIFICATE_FILE: "/etc/letsencrypt/live/{{ ansible_fqdn }}/chain.pem"
-RABBITMQ_CERTIFICATE_FILE: "/etc/letsencrypt/live/{{ ansible_fqdn }}/cert.pem"
-RABBITMQ_KEY_FILE: "/etc/letsencrypt/live/{{ ansible_fqdn }}/privkey.pem"
+RABBITMQ_CA_CERTIFICATE_FILE: "/etc/letsencrypt/live/{{ inventory_hostname }}/chain.pem"
```

Comment:
I suggest introducing a variable `RABBITMQ_HOSTNAME` which defaults to `inventory_hostname`.  We've used the same pattern in other places, and it sometimes makes things easier during migrations if you have the ability to temporarily set this to a different value.